tests/test_getting_started.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -eo pipefail
+
+# Step 1: Python environment
+# Check Python version is or later than 3.9
+echo "Step 1: Python environment"
+echo "Checking Python version which should be equal or later than 3.9"
+if ! python -c 'import sys; assert sys.version_info >= (3,9)' > /dev/null; then
+    exit "Python should be 3.9 or later!"
+fi
+
+# Clone repo
```

Comment:
@Deegue It looks to me, there are some duplications for getting_started and setup on the setup process, is it better we can consolidate them into single one?

Reply:
`test_getting_started.sh` aims to test the basic functions under CPU, while `test_setup.sh` which may be tested on docker, can accept configs like CPU, GPU and Gaudi with or without Deepspeed.

Reply:
> `test_getting_started.sh` aims to test the basic functions under CPU, while `test_setup.sh` which may be tested on docker, can accept configs like CPU, GPU and Gaudi with or without Deepspeed.

I see. let us do it your way right now.